### ISSUE TITLE
Customized / Personalized soundtherapy for tinnitus (based of personal tinnitus profile)

### ISSUE THREAD

**Issue by huuhaa** [ashutoshgngwr/noice#794]
### Is your feature request related to a problem? Please describe.

Sound therapy for tinnitus is known to be effecting help, eventhought it's not gonna cure tinnitus. More effective form of sound therapy is custom / personalized sound therapy which uses persons tinnitus profile to offering right kind of background noice which user is supposed to listen an hour a day (or that's at least what I've been read. Claim for that being most effective is simply cause one size doesn't fit all, best noice for tinnitus is based for person tinnutus profile.

I don't know how those tinnitus profiles ar created, might be sum of hearing test + some selection to descripe what kind of tinnitus person hears? If the could be some way to ask user to describe tinnitus, maybe that questionnare could result generating better fitting sound therapy as user does by quessing. Sure it would be cool if actual hearing test would be possible to make too, or at lest using somehow results of official hearing test.

Tinnitus profile is term used at some Finnish audiology sites, but seems like english pages (like beyondtinnitus etc.) are talking more just custom sound therapy or some case personalized.

### Describe the solution you'd like
I would like to have option under quick function to autogerate preset, to select to generate preset for tinnitus. When that selected, there could be some "questionnaire" to tick boxes which describes tinnitus best. Sure it would also need some scientic based information for how to generate matching preset, so probably not easy task.

### Additional notes

There are different kind of tinnitus, from wikipedia: often described as a ringing, it may also sound like a clicking, buzzing, hiss, or roaring. The sound may be soft or loud, low or high pitched, and often appears to be coming from one or both ears or from the head itself.

So it could be example like hearing multiple high piched noices from one or both ear, just one noice, hiss etc. Could be chronical or temporarily. So how could user give input to provide all this information.

https://en.wikipedia.org/wiki/Tinnitus#Sound-based_interventions

### Ending notes

I do understand, that it might be impossible task to try to implement feature like this for open source app, since it most likely needs knownledge of experts which might or might not to want to participate and is most likely very difficult task anyway.

But since you can't achieve anything unless you at lest try, I decided that I will leave this feature request for you to make your decisions about this. Now this is at least suggested.

**Comment by ashutoshgngwr** [comment]
I read the research articles referenced from the Wikipedia section on Tinnitus sound therapy. Personalised sound therapy has no basis in Noice's domain. The sounds used are completely different from the ones you conventionally find in Noice, e.g. the following is an excerpt describing a sound therapy device from [one of the articles](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=+%5BPDF%5D+academia.edu+Sound+therapy+for+tinnitus+management%3A+practicable+options&btnG=).

> This device offers sound generator options of broadband signals, a narrowband signal focused on the frequency of the tinnitus, and the option of using amplitude modulation (AM; fluctuation in the level of noise signal while all other spectral components remain uniform).

A non-clinical app can only offer partial masking at best. The sounds used in apps like Noice are incapable of offering anything better. Partial masking can only help distract patients from the tinnitus sound and doesn't offer any long-term benefits. The same article describes sound therapy as the following.

> Sound therapy is defined in the present review as any use of sound where the intention is to alter the tinnitus perception and/or the reactions to tinnitus in a clinically meaningful way.

So to summarise, my thoughts are as follows.

- The domain overlap between Noice and Tinnitus is partial masking only. Partial masking isn't considered sound therapy since it doesn't offer clinical benefits to patients.
- As far as I read, I didn't come across any research that supports the personalisation of partial masking.
- Most of the sound therapy options require a _prescription_ from an audiologist.

I am closing this issue. I think it is safe to say that it's _(mostly)_ outside our domain. Something like this requires an entire project of its own. It can't be sideloaded in apps like Noice.
